src/services/item/plugins/app/service.ts

```diff
@@ -1,16 +1,19 @@
 import uniqBy from 'lodash.uniqby';
 
-import { ItemType, PermissionLevel, UUID } from '@graasp/sdk';
+import { AuthTokenSubject, ItemType, PermissionLevel, UUID, UnionOfConst } from '@graasp/sdk';
 
 import { Repositories } from '../../../../utils/repositories';
 import { validatePermission } from '../../../authorization';
 import { Actor, Member } from '../../../member/entities/member';
-import { Item } from '../../entities/Item';
+import { Item, isAppItem } from '../../entities/Item';
 import { checkTargetItemAndTokenItemMatch } from './utils';
 
 export class AppService {
   jwtExpiration: number;
-  promisifiedJwtSign: any; // TODO
+  promisifiedJwtSign: (
+    arg1: { sub: AuthTokenSubject },
+    arg2: { expiresIn: string },
```

Comment:
Should these parameters have more descriptive names? E.g. the original parameter names from `jwtSign`